.github/styles/Vocab/FluxNinja/accept.txt

```diff
@@ -50,6 +50,20 @@ Prometheus
 Pub/Sub
 [Aa]uto-scalers?\b
 [Rr]ollouts?\b
+LinkedIn
+Nutanix
+Netsil
+SLAs
+end-user
+rpm
+performant
+Zalando
+Hotstar
+Linkerd
+multithreaded
+look at
 AIMD
```

Comment:
Why should this be allowed? Rephrase the sentences that use this in a sentence

Reply:
Done!